### ISSUE TITLE
Accessibility for virtual scrolling

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#2426]
## Description
After the current implementation of the virtual scroll will be merged (#2280) we'll be missing accessibility for this. Needs to be implemented

## Possible solutions:
- Visually hidden pagination

**Comment by PVince81** [comment]
From what I remember in our discussion, the pagination in question would also need navigation buttons, like back in the old days of the web. Page numbers and arrows.

**Comment by PVince81** [comment]
cc @marcus-herrmann

**Comment by PVince81** [comment]
If not possible to make the Phoenix table accessible, we might need to have a "mode of operation" that the user can configure to tell whether they want to use pagination or virtual scroll.

Ideal would be to have all this packaged already inside the "oc-table" element to prevent having to use different elements in Phoenix.

**Comment by marcus-herrmann** [comment]
There are two broad accessibility issues with infinite scrolling, aside from the container (table or div)

# For keyboard only users
Since keyboard only users don't use landmarks (`<header>`, `<main>`, `<footer>` and the like) they won't ever get past the infinite scrolling construct in the dom (Usually: focus order = DOM order). The suggestion of a pagination wasn't meant to be exclusively for screen reader users, therefore a visually hidden pagination would be hidden for keyboard only users.

# For screen reader users
`role="feed"` does exist, but is no silver bullet [(Source)](https://www.deque.com/blog/infinite-scrolling-rolefeed-accessibility-issues/)

> ...while infinite scroll becomes accessible to screen reader users in browse/reading mode, only those are very comfortable with screen readers (or power users) will be able to identify these widgets. Some less experienced screen reader users might think the page is very long and might miss the complementary/footer region completely

I'm in favor of offering a choice for all, like this: http://a11ytest.dequelabs.com/preview.aspx?ID=MzI5#

**Comment by LukasHirt** [comment]
We've decided to go with pagination instead of a virtual scroll.
